i18n/data/zh-CN.json

```diff
@@ -15,10 +15,10 @@
   "UIKit.CreatableSelectInput.noOptionsMessageWithInputValue": "无选项",
   "UIKit.CreatableSelectInput.noOptionsMessageWithoutInputValue": "无选项",
   "UIKit.CreatableSelectInput.placeholder": "选择...",
-  "UIKit.DateInput.placeholder": "月/日/年",
-  "UIKit.DateRangeInput.placeholder": "月/日/年 - 月/日/年",
-  "UIKit.DateTimeInput.placeholder": "月/日/年 - 小时：分钟 上午/下午",
-  "UIKit.DateTimeInput.timePlaceholder": "小时：分钟 上午/下午",
+  "UIKit.DateInput.placeholder": "年/月/日",
```

Comment:
moment formats chinese dates to Years/Month/Day.

https://github.com/moment/moment/blob/develop/locale/zh-cn.js